app/routes/pages/components/PageDetail.css

```diff
@@ -80,3 +106,45 @@
   justify-content: center;
   flex-direction: column;
 }
+
+.sidebarOpenBtn {
+  background-color: #f7f7f7;
+  width: 40px;
+  height: 65px;
+  position: -webkit-sticky; /* Safari */
+  position: sticky;
+  top: 0;
+  display: none;
+  align-items: center;
+  justify-content: center;
```

Comment:
Does `align-items` and `justify-content` do anything when `display: none` ?

Reply:
I guess these styles should be moved to the mobile css with same class name, since this applies only on mobile where the display attribute is changed.

Reply:
Really, if that's the case, everything should be moved to the mobile except for display none.